model/metrics/clr-metrics.yaml

```diff
@@ -0,0 +1,167 @@
+groups:
+  - id: metric.clr.gc.collections.count
+    type: metric
+    metric_name: clr.gc.collections.count
```

Comment:
Duration would be useful, but I think would require the implementation to use an `EventListener` for the [GC events](https://learn.microsoft.com/en-us/dotnet/fundamentals/diagnostics/runtime-garbage-collection-events). I [raised this](https://github.com/dotnet/runtime/issues/85372#issuecomment-2114531323) as a possibility in the runtime issue, but @noahfalk did [point out](https://github.com/dotnet/runtime/issues/85372#issuecomment-2118494819) that this may have additional overhead.